### ISSUE TITLE
Restore attribution tweak cannot be disabled

### ISSUE THREAD

**Issue by marcustyphoon** [AprilSylph/XKit-Rewritten#1087]
### Platform

MacOS 13.2.1 arm64

### Browser

Chrome 113.0.5672.126

### Addon version

12dcad4

### Details

The new tweak to make post header links point to individual posts does not turn itself off when it or the entire Tweaks script is disabled, as it does not have a fully implemented `clean()` function.

This is pretty minor, of course, but it's reasonably easy to implement (e.g. storing the original href value in a data attribute).

**Comment by inklesspen** [comment]
Yeah, I noted that in the PR (#1084); I figured if anyone truly cared about having the links go back, they can refresh the page.
